cfgov/v1/atomic_elements/organisms.py

```diff
@@ -24,7 +24,6 @@
 from v1.atomic_elements.tables import AtomicTableBlock
 from v1.util import ref
 
-
```

Comment:
And two blank lines are required before classes so add this line back to get the linter to pass.